Make vis-lint more configurable

rnpnr opened the pull request:
This will apply on top of #2 once it is merged and will add a number of configuration options to the plugin.

rnpnr committed:
make linters and fixers configurable

rnpnr committed:
return false only on command failure

rnpnr committed:
export lint and fix as functions

roguh commented:
I recommend mentioning that commands must be compatible with the  stdout/stdin streams. Perhaps show some examples of how to convert a command into that form? E.g.

> try `awkfix -` or `awkfix <(cat)` or simply `awkfix` or check if the tool has other options to accept input via stdin, such as `awkfix --from-stdin`

rnpnr commented:
I added a comment about using `stdin`/`stdout` in the README. I'm not keen on mentioning anything to do with `<(...)` since its a pretty much only supported by bash. The default shell in `vis` is `/bin/sh` which in many cases is not `bash`.

I also switched the example to use `pylint` instead of `flake`. In this case I don't think having more defaults is better. I think 2 is sufficient to demonstrate that you can use more than 1. I am willing to reconsider though if there is a compelling reason.

In any case I'm going to merge both pull requests.
